blaze/server/tests/test_server.py

```diff
@@ -369,3 +344,28 @@ def test_serialization_endpoints(serial):
     result = respdata['data']
     assert result == odo(compute(expr, {t: data}), int)
     assert respdata['names'] == ['petal_length_sum']
+
+
+@pytest.mark.parametrize('serial', all_formats)
+def test_cors_compute(test, serial):
```

Comment:
@fpliger I couldn't get this test to fail. It passes whether I have CORS enabled or not. Is there something I'm not doing properly to get it to fail when I *haven't* enabled CORS?